inc/saiport.h

```diff
@@ -2025,6 +2093,21 @@ typedef enum _sai_port_stat_t
     /** PRBS Error Count */
     SAI_PORT_STAT_PRBS_ERROR_COUNT,
 
+    /** SAI port stat if in FEC correctable pkts */
+    SAI_PORT_STAT_IF_IN_FEC_CORRECTABLE_FRAMES,
+
+    /** SAI port stat if in FEC not correctable pkts */
+    SAI_PORT_STAT_IF_IN_FEC_NOT_CORRECTABLE_FRAMES,
+
+    /** SAI port stat if in FEC symbol errors */
+    SAI_PORT_STAT_IF_IN_FEC_SYMBOL_ERRORS,
```

Comment:
are these stats SAI_PORT_STAT_IF_IN_FEC_* applicable for fabric port or any reglar port ?

Reply:
The *FEC* stats are applicable to fabric ports in this VoQ context but can also apply to front panel ports as well.